Makefile.one

```diff
@@ -0,0 +1,442 @@
+.PHONY: all
+all:
+	$(MAKE) env
+	$(MAKE) reqs
+	$(MAKE) tracee-ebpf tracee-rules rules
+
+# make
+
+PARALLEL = $(shell $(CMD_GREP) -c ^processor /proc/cpuinfo)
+
+NORM_MAKE = make -j$(PARALLEL)
+
+MAKE = make -f Makefile.one
+MAKEFLAGS += --no-print-directory
+MAKEFLAGS += -j$(PARALLEL)
+
+# tools
+
+CMD_GIT ?= git
+CMD_CLANG ?= clang
+CMD_LLC ?= llc
+CMD_STRIP ?= llvm-strip
+CMD_RM ?= rm
+CMD_INSTALL ?= install
+CMD_MKDIR ?= mkdir
+CMD_PKGCONFIG ?= pkg-config
+CMD_GO ?= go
+CMD_GREP ?= grep
+CMD_OPA ?= opa # https://github.com/open-policy-agent/opa/releases/download/v0.33.1/opa_linux_amd64
+
+check_%:
+	@command -v $* >/dev/null || (echo "missing required tool $*" ; exit 1)
+
+# libs
+
+LIB_ELF ?= libelf
+LIB_ZLIB ?= zlib
+
+checklib_%:
+	@$(CMD_PKGCONFIG) --silence-errors --validate $* 2>/dev/null || (echo "missing lib $*" ; exit 1)
+
+# version
+
+LAST_GIT_TAG ?= $(shell $(CMD_GIT) describe --tags --exclude 'tracee-rules/*' --exclude 'tracee-ebpf/*' 2>/dev/null)
```

Comment:
maybe a safer way is to `--include 'v*'`

Reply:
--match 'v*' works.